flycheck.el

```diff
@@ -8681,6 +8685,17 @@ Relative paths are relative to the file being checked."
     (with-output-to-string
       (call-process "rustc" nil standard-output nil "--explain" error-code))))
 
+(defun flycheck-rust--verify-cargo-toml ()
+  "Return a flycheck-verification-result indicating whether Cargo.toml was found."
+  (let ((have-cargo-toml (locate-dominating-file (buffer-file-name) "Cargo.toml")))
+    (flycheck-verification-result-new
+     :label "Cargo.toml"
+     :message (if have-cargo-toml "found" "missing")
+     :face (if have-cargo-toml 'success '(bold error))
+     )
+    )
+  )
```

Comment:
Please do not use hanging parenthesis.  Instead, move all the closing parenthesis up to the last code line.

I know it feels odd coming from other languages, but in Lisp it really aids readability to keep parenthesis along with the code.